src/commands/compile-go.yml

```diff
@@ -8,6 +8,9 @@ steps:
   - run:
       name: Go compile
       command: |
+        go env -w GOPRIVATE=$GOPRIVATE
+        git config --global --unset url."ssh://git@github.com".insteadOf
```

Comment:
is an `insteadof` needed here? there is no arg provided to insteadof

Reply:
Yes, it is an option which we need to unset.